includes/services.inc.php

```diff
@@ -129,7 +129,7 @@ function poll_service($service)
 
     // If we do not have a cmd from the check script, build one.
     if ($check_cmd == "") {
-        $check_cmd = $config['nagios_plugins'] . "/check_" . $service['service_type'] . " -H " . ($service['service_ip'] ? $service['service_ip'] : $service['hostname']);
```

Comment:
@algebur Two suggestions I see this all the time with people wanting to have devices show up as names but add them as an IP.  There are two options.

1. Set up a DNS server, even if it is just a local one on the LibreNMS box, but then you won't get the full benefit of the work to set it up.
2. Use this setting `$config['force_ip_to_sysname'] = true;`  If you see a spot that should respect that setting, but it does not, I suggest adding support.

P.S. https://github.com/librenms/librenms/blob/master/html/includes/table/alerts.inc.php